src/views/Orders.vue

```diff
@@ -212,6 +226,26 @@ export default defineComponent({
       });
       return batchmodal.present();
     },
+    async deleteBatch(batch: any) {
+      const deleteBatchAlert = await alertController
+        .create({
+          header: this.$t('Cancel job'),
+          message: this.$t("Canceling this job will cancel this occurrence and all following occurrences. This job will have to be re-enabled manually to run it again."),
+          buttons: [
+            {
+              text: this.$t("Don't cancel"),
+              role: 'cancel',
+            },
+            {
+              text: this.$t("Cancel"),
+              handler: async () => {
+                  await this.store.dispatch('job/cancelJob', batch);
```

Comment:
Improve indentation to one tab/two space.

Reply:
Done sir!
Please review - https://github.com/hotwax/job-manager/pull/140/commits/989d77e24f83a00ba4ec6ff6ae5ad50dcef77bf0